sql/core.go

```diff
@@ -358,6 +358,10 @@ type PrimaryKeyAlterableTable interface {
 	CreatePrimaryKey(ctx *Context, columns []IndexColumn) error
 	// DropPrimaryKey drops a primary key on a table. Returns an error if that table does not have a key.
 	DropPrimaryKey(ctx *Context) error
+	// Pks returns a list of ordered primary key columns
+	Pks() []IndexColumn
+	// PrimaryKeySchema returns this table's PrimaryKeySchema
+	PrimaryKeySchema() PrimaryKeySchema
```

Comment:
As mentioned elsewhere, this is defined on the wrong interface. Make a new one, `PrimaryKeyTable`